src/toast/ToastProvider.tsx

```diff
@@ -26,11 +27,13 @@ interface ToastContextProviderProps {
 function ToastContextProvider({
   children,
   customRootId,
-  autoCloseToasts = true
+  autoCloseToasts = true,
+  limit
 }: ToastContextProviderProps) {
   const [state, dispatch] = useReducer(toastReducer, {
     ...initialToastState,
-    autoCloseToasts
+    autoCloseToasts,
+    limit
   });
 
```

Comment:
Can you add an effect here that listens the changes on `limit` prop and reflects the changes in `limit` within the state later on ? Same thing for the `autoCloseToasts` prop. The users of the toast provider may have a need to change the limit or the autoCloseToasts dynamically in the lifetime of an app, e.g. limit may change when the screen size is small.

Reply:
@mfarukkoc Looks like you made the changes Ediz mentioned. After making changes mentioned in a comment, you can "Resolve conversation" using the button below.